incubator/virtualcluster/pkg/syncer/patrol/differ/handler.go

```diff
@@ -86,19 +86,20 @@ func (h FilteringHandler) OnDelete(obj ClusterObject) {
 	h.Handler.OnDelete(obj)
 }
 
-func DefaultDifferFilter(blockedClusterSet sets.String) func(obj ClusterObject) bool {
+func DefaultDifferFilter(knownClusters []string) func(obj ClusterObject) bool {
```

Comment:
left the input parameter the buildin type `[]string`
`sets` is the Second-class citizens